docker-compose.yml

```diff
@@ -7,6 +7,7 @@ x-env-mapping: &env
     - CELERY_RESULT_BACKEND=redis://redis:6379/1
     - DATABASES_DEFAULT_URL=mysql://root:@mysqld/olympia
     - ELASTICSEARCH_LOCATION=elasticsearch:9200
+    - ELASTIC_CLIENT_APIVERSIONING=1
```

Comment:
We'll need SRE to add that to our addons-server dev/stage/prod environments as well before they can switch the cluster for those environments to 8.x.

Reply:
Will you file a follow up ticket or what's the plan?

Reply:
Thinking about this more... We should have that in the `Dockerfile` - and once all envs are on 8.x server, when we upgrade the client libs, we'll remove it.

Reply:
https://github.com/mozilla/addons-server/pull/23299/commits/e902122f8a156384f38db3633729d5a8d24ac55d